config/_default/menus/main.en.yaml

```
@@ -5125,7 +5125,7 @@ menu:
       parent: appsec_enabling_single_step
       identifier: appsec_enabling_single_step_code
       weight: 2102
-    - name: Using Datadog Tracing libraries
+    - name: Using Datadog Tracing Libraries
```

We don't want to change this one?
```suggestion
    - name: Using Datadog APM SDKs
```

---

Thanks for catching. This should be fixed now.